axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/batch/BatchDirectDebitPaymentSchedule.java

```diff
@@ -95,7 +101,11 @@ protected void processPaymentScheduleLines(int paymentScheduleType) {
           companyBankDetails.getCurrency(),
           I18n.get("Currency in company bank details is missing."));
       Preconditions.checkArgument(
```

Comment:
Create variable for accountingBatch.getPaymentMode() and use everywhere in method

Create new  Preconditions.checkArgument above this line to check Strings.isNullOrEmpty(paymentMode.getBankOrderExportFolderPath()) with same error message